llvm/lib/DebugInfo/LogicalView/Core/LVScope.cpp

```diff
@@ -1011,9 +1011,13 @@ void LVScope::printExtra(raw_ostream &OS, bool Full) const {
   // Do not print any type or name for a lexical block.
   if (!getIsBlock()) {
     OS << " " << formattedName(getName());
-    if (!getIsAggregate())
+    if (!getIsAggregate()) {
       OS << " -> " << typeOffsetAsString()
          << formattedNames(getTypeQualifiedName(), typeAsString());
+    }
+    if (uint32_t Size = getStorageSizeInBytes();
+        options().getAttributeSize() && Size)
```

Comment:
May be rearrange the conditions to call `getStorageSizeInBytes()` only when `options().getAttributeSize()` is true `(--attribute=size)`.

Reply:
Done :+1:; PTAL.